app/controllers/api/v1/keywords_controller.rb

```diff
@@ -4,10 +4,8 @@ module Api
   module V1
     class KeywordsController < ApplicationController
       def index
-        first_keyword = Keyword.new({ id: 1, name: 'First keyword' })
-        second_keyword = Keyword.new({ id: 2, name: 'Second keyword' })
-
-        render json: KeywordSerializer.new([first_keyword, second_keyword]).serializable_hash.to_json
+        keywords = Keyword.where(user_id: current_user.id)
```

Comment:
Can we 💭 ? If possible i prefer passing an object to the Active query.

```suggestion
        keywords = Keyword.where(user: current_user)
```